### ISSUE TITLE
Code assist for callbacks in an operation creates an invalid object

### ISSUE THREAD

**Issue by tfesenko** [RepreZen/KaiZen-OpenAPI-Editor#320]
Reported by @daffinm 

Invoke code assist to create a callback inside an operation, it will suggest "[a-zA-Z0-9\.\-_]+$", which is invalid:
<img width="417" alt="screen shot 2017-05-24 at 10 14 53 am" src="https://cloud.githubusercontent.com/assets/644582/26407865/45e45380-406a-11e7-923b-33d29ff7b87a.png">
Code assist for callbacks inside components is OK: it suggests a "_key_" value or a code template
<img width="386" alt="screen shot 2017-05-24 at 10 18 44 am" src="https://cloud.githubusercontent.com/assets/644582/26407914/68673526-406a-11e7-92ba-b2b8c9febdf3.png">

This issue is similar to #310 - another example of code assist proposals based on pattern properties - but this case is more severe as it does not provide a code template.

**Comment by tedepstein** [comment]
@daffinm , you also reported this issue in the ZenDev slack channel this morning.

**Comment by tedepstein** [comment]
Sorry @daffinm , ignore the above comment @tfesenko , already mentioned that she created this issue from your bug report in slack.

**Comment by ghillairet** [comment]
Could that be an issue with the schema, a callback is defined like that:

```json
"callbacks": {
      "type": "object",
      "description": "A map of possible out-of band callbacks related to the parent operation. Each value in the map is a Callback Object that describes a request that may be initiated by the API provider and the expected responses. The key value used to identify the callback object is an expression, evaluated at runtime, that identifies a URL to use for the callback operation.",
      "additionalProperties": false,
      "patternProperties": {
        "^[a-zA-Z0-9\\.\\-_]+$": {
          "$ref": "#/definitions/callbackOrReference"
        },
        "^x-": {
          "$ref": "#/definitions/specificationExtension"
        }
      }
    }
```

**Comment by ghillairet** [comment]
Fixed in #330 

![callbacks](https://cloud.githubusercontent.com/assets/148045/26488469/cb6e01c8-4203-11e7-9733-5f558d2fa128.gif)
